volatility3/framework/plugins/windows/shimcachemem.py

```diff
@@ -146,7 +146,7 @@ def find_shimcache_win_xp(
             context, layer_name, kernel_symbol_table
         ):
             pid = process.UniqueProcessId
-            vollog.debug("checking process %d" % pid)
+            vollog.debug("checking process %d", pid)
```

Comment:
Errr, will that fill in the `%d` or will it just have `%d <pid>` in the text?

Reply:
AFAIK this works

```python
>>> import logging
>>> logger = logging.getLogger(__name__)
>>> logger.warning("example %d", 12345)
example 12345
```
pretty sure .debug works the same

Reply:
Ok, cool, happy with it then, thanks for testing it.  5:)